client/src/app/core/ui-services/poll.service.ts

```diff
@@ -1,4 +1,5 @@
 import { Injectable } from '@angular/core';
+import { PollVoteValue } from 'app/shared/models/assignments/poll-option';
```

Comment:
Please do not import anything from shared here. So move `PollVoteValue` here, or make a new file in the core for this.